lighthouse-cli/test/fixtures/seo/seo-tester.html

```diff
@@ -43,11 +51,14 @@ <h2>Anchor text</h2>
 
   <h2>Small text</h2>
   <!-- PASS(font-size): amount of illegible text is below the 60% threshold -->
-  <p class='small'> 1 </p>
-  <p class='small-2'> 2 </p>
+  <!-- font-size items are ordered by text length, so force an order with filler for stable expectations. -->
+  <p class='small'> 1.... </p>
+  <p class='small-2'> 2... </p>
+  <p class='small-3'> 3.. </p>
```

Comment:
adding these two cases made the actual `items` jump around in `font-size`, on account of most items being the same text length and the audit using an unstable sort on that property. I added filler text to maintain the order.